misc/python/materialize/zippy/kafka_actions.py

```diff
@@ -70,7 +74,7 @@ def requires(cls) -> Set[Type[Capability]]:
     def __init__(
         self,
         max_topics: int = 10,
-        envelopes: List[Envelope] = [Envelope.NONE, Envelope.UPSERT],
+        envelopes: Dict[Envelope, int] = {Envelope.NONE: 25, Envelope.UPSERT: 75},
```

Comment:
Optional: rename to `envelopes_with_weights`

Reply:
Thank you, updated.